pyv/transpiler.py

```diff
@@ -222,8 +222,15 @@ def visit_Call(self, node: ast.Call) -> str:
 
         vargs: List[str] = []
 
-        if node.args:
-            vargs.extend(map(self.visit, node.args))
+        for idx, arg in enumerate(node.args):
+            if (
+                isinstance(arg, ast.Name)
+                and isinstance(fndef, ast.FunctionDef)
+                and is_mutable(fndef.scopes, fndef.args.args[idx].arg)
+            ):
+                vargs.append(f"mut {arg.id}")
```

Comment:
`get_id(arg)` is safer.
Also `hasattr(fndef, "args")`.

Reply:
get_id makes more sense, but is there any case in which fndef does not have args? According to the latest Python AST documentation, it seems to be not optional.

Reply:
I think it breaks when using lambdas. The check was introduced in pyrs via 6861cca54e0ae95f194a7d00af43d98b38a7b97b
I just tested that removing the check and running

t -k "rust and ambda" -v

causes a test breakage.